File: tests/test-minecraft/src/main/java/pize/tests/minecraft/game/gui/text/formatting/ColorFormatting.java
package pize.tests.minecraft.game.gui.text.formatting;

import pize.graphics.util.color.ImmutableColor;

public class ColorFormatting implements ITextFormatting{

    private final ImmutableColor color;

    public ColorFormatting(int r, int g, int b){
        this.color = new ImmutableColor(r, g, b, 255);
    }

    public ImmutableColor getColor(){
        return color;
    }

}

File: tests/test-minecraft/src/main/kotlin/pize/tests/minecraft/game/gui/text/formatting/ColorFormatting.kt
package pize.tests.minecraft.game.gui.text.formatting

import pize.graphics.util.color.ImmutableColor

class ColorFormatting(r: Int, g: Int, b: Int) : ITextFormatting {
    val color: ImmutableColor

    init {
        color = ImmutableColor(r, g, b, 255)
    }
}
